python/src/rest-api/rest_api.aac

```diff
@@ -0,0 +1,55 @@
+plugin:
+  name: REST API
+  package: rest_api
+  description: |
+    An AaC plugin that provides a RESTful API for managing and interacting with
+    the AaC Python package.
+  commands:
+    - name: rest-api
+      help_text: Start a RESTful interface for interacting with and managing AaC.
+      run_before:
+        - plugin: Check AaC
+          command: check
+      run_after:
+        - plugin: Generate
+          command: generate
+      input:
+        - name: --host
+          type: string
+          description: Set the hostname of the service. Useful for operating behind proxies.
+          default: "0.0.0.0"
+        - name: --port
+          type: int
+          description: The port to which the RESTful service will be bound.
+          default: "8080"
+      acceptance:
+        - name: Start the RESTful API.
+          scenarios:
+            - name: Start the RESTful API without arguments.
+              when:
```

Comment:
Missing the `given` step

Reply:
This scenario is for the default configuration, so that should be what it focuses on evaluating.

Reply:
Given is not a required field, and since it is the default config I don't think it needs one.  It uses the default port 8080.